modules/sbo-deploying-the-spring-petclinic-sample-application.adoc

```diff
@@ -10,81 +10,81 @@ To deploy the Spring PetClinic sample application on an {product-title} cluster,
 . Deploy the `spring-petclinic-rest` application with the `PostgresCluster` custom resource (CR) by running the following command in shell:
```

Comment:
Shouldn't this be:  Deploy the `spring-petclinic` apllication...

Reply:
Thanks, @SNiemann15, good catch! :)